packages/esm-ward-app/src/ward-view/ward-view.resource.ts

```diff
@@ -12,3 +12,10 @@ export function bedLayoutToBed(bedLayout: BedLayout): Bed {
     status: bedLayout.status,
   };
 }
+
```

Comment:
Can we make this a generic method in esm-api that takes in a moduleId and returns true if that module is installed?